eng/pipelines/runtime-extra-platforms-other.yml

```diff
@@ -72,6 +72,35 @@ jobs:
           eq(dependencies.evaluate_paths.outputs['SetPathVars_libraries.containsChange'], true),
           eq(variables['isRollingBuild'], true))
 
+#
+# CoreCLR NativeAOT release build and additional libraries tests
+# Only when CoreCLR or library is changed
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    helixQueuesTemplate: /eng/pipelines/libraries/helix-queues-setup.yml
+    buildConfig: Release
+    platforms:
+    - windows_x64
+    - windows_arm64
+    jobParameters:
+      testGroup: innerloop
+      isSingleFile: true
+      nameSuffix: NativeAOT_Additional_Libs
+      buildArgs: -s clr.alljits+clr.tools+clr.nativeaotlibs+clr.nativeaotruntime+libs+libs.tests -c $(_BuildConfig) /p:TestNativeAot=true /p:TestNativeAotAdditionalLibs=true /p:ArchiveTests=true
+      timeoutInMinutes: 180
+      # extra steps, run tests
+      extraStepsTemplate: /eng/pipelines/libraries/helix.yml
+      extraStepsParameters:
+        creator: dotnet-bot
+        testRunNamePrefixSuffix: NativeAOT_$(_BuildConfig)
```

Comment:
Would you want the testRunNamePrefixSuffix to also be "NativeAOT_Additional_Libs" or will the nameSuffix already show that?

Reply:
Not sure, hopefully the new pipeline run will make it clearer.

Reply:
The extra-platform pipeline completed and the Job name description, "Build windows x64 Release NativeAOT_Additional_Libs", seems sufficient to me. Let me know if you want testrunname to be changed as well.